cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/LocalDataLayer.java

```diff
@@ -174,7 +174,7 @@ public static LocalDataLayer from(Map<String, String> options)
                       .filter(StringUtils::isNotEmpty)
                       .collect(Collectors.toSet()),
                 SchemaFeatureSet.initializeFromOptions(options),
-                getBoolean(options, lowerCaseKey("useSSTableInputStream"), false),
+                getBoolean(options, lowerCaseKey("useBufferingInputStream"), false),
```

Comment:
Is anything going to break because the name of this option changed? Should there be compatibility with the old name?

Reply:
The `LocalDataLayer` is mainly for the tests to read/write SSTables, but I can maintain compatibility with the old name incase anyone is using it in the wild.